common/views/components/Templates/BasicPage/BasicPage.js

```diff
@@ -12,8 +12,8 @@ type Props = {|
   title: string,
   body: Body,
   mainImageProps: ?UiImageProps,
+  Background: ?(| WobblyBackground),
```

Comment:
can you explain this notation to me – I'm confused.

Reply:
It was used to hint that this will become a union type, but is\t actually that useful.
I;ve removed it.